python/ml4ir/model/ranking_model.py

```diff
@@ -8,10 +8,12 @@
 from tensorflow import TensorSpec, TensorArray
 from tensorflow import data
 from tensorflow.keras import metrics as kmetrics
+from tensorflow.keras import backend as kbackend
```

Comment:
don't people often abbreviate this as simply `K`, conventionally?

`from tensorflow.keras import backend as K`

Not sure why... but seems standardish

Reply:
Yeah, good call.